corehq/apps/locations/views.py

```diff
@@ -1050,7 +1050,7 @@ def loc_to_payload(loc):
     paginator = Paginator(locs, 10)
     return json_response({
         'results': list(map(loc_to_payload, paginator.page(page))),
-        'total_count': paginator.count,
+        'total': paginator.count,
```

Comment:
This was originally added in https://github.com/dimagi/commcare-hq/pull/18794/files but i think at some point the name was changed in JS but not in the view. I searched quite a bit to ensure that this view is not used anywhere else, so updated in the view directly. If someone else can quickly check that too, that would give more confidence in this change. No pressure though.

Reply:
I double-checked, looks good.

Reply:
thank you @orangejenny